roadmap/EF2025.md

```diff
@@ -0,0 +1,114 @@
+for an overview).
+
+Throughout the year we have done additional work to make the interfaces within the libraries more tree-agnostic (so: that it does not matter
+if a Merkle or Verkle tree is used) and brought our own [verkle tree implementation](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/verkle) to the point where we now can do stateful verkle execution.
+
+Together with the new test generation capabilities for the [cross client tests](https://github.com/ethereum/execution-spec-tests/releases/tag/verkle%40v0.0.5) in `execution-spec-tests` we have gained (see "Protocol Testing" track) we have now everything together to support Verkle development on a
+broader level.
+
+### 2025 Deliverables
+
+- Continued early-on spec implementations and feedback
+- Help setup and run stateless verkle testnets
+- Continuous verkle test generation for [ethereum-execution-specs](https://github.com/ethereum/execution-specs)
+- Active contributions to work on verkle test coverage and fill tests
+
+## Stateless Verkle Client & Modularization
+
+Track responsible: [Amir Ghorbanian](https://github.com/scorbajio)
+
+Stateless will make (parts of) a browser based/used client more viable. We have all the building blocks together (Verkle, VM, client) to serve software
+components for this use case. 2025 will be the year where we can bundle all existing pieces together respectively refactor in a way so that browser
+applications will be able to integrate lightweight verkle block verification into their code base. This will allow developers to build their dApps in a
+more trustworthy way by rely on less 3rd party trust assumptions.
+
+### 2025 Deliverables
+
+- Modularization of the EthereumJS client code
+- App -> Library: reusage/integration into existing code bases
+- Strong client dependency reduction to prepare for production use cases
+- Verkle-tailored stateless client version of the client
+- Work on outer UX experience
+
+## Portal Network & EIP-4444
+
+Track resonsible: [Scotty Poi](https://github.com/ScottyPoi)
+
+[Portal network](https://github.com/ethereum/portal-network-specs) evolved throughout 2024 and is now at a point where the specs matured for early-developed networks like the history network and is now considered for production usage e.g. in the form of being a source for providing chain history in an [EIP-4444](https://eips.ethereum.org/EIPS/eip-4444) context. The JavaScript team is developing its own [Ultralight](https://github.com/ethereumjs/ultralight) portal client and contributed to Portal development with an early on state network implementation (see e.g. PRs [#539](https://github.com/ethereumjs/ultralight/pull/539) and [#649](https://github.com/ethereumjs/ultralight/pull/649)) as well as providing robust bootnodes for setting up test networks, among other things.
```

Comment:
I would be hesitant to say our bootnodes are "robust" unless @ScottyPoi signs off on it.  :grimacing:

Reply:
Have now fully used Scotty's version